reference-architectures/watsonx-saas-da.md

```diff
@@ -0,0 +1,148 @@
+the toc attributes on the H1, repeating the values from the YAML header.
+ -->
+
+The Watsonx AI SaaS deployable architecture is designed to automate the deployment and
+configuration of the {{site.data.keyword.IBM_notm}} watsonx platform in an {{site.data.keyword.Bluemix_notm}} account. The {{site.data.keyword.IBM_notm}} watsonx platform is made of several services working together to offer AI capabilities to end users who can explore them using [{{site.data.keyword.IBM_notm}} watsonx projects](https://dataplatform.cloud.ibm.com/docs/content/wsj/manage-data/manage-projects.html?context=wx&audience=wdp). The automation also configures a starter project for an existing {{site.data.keyword.Bluemix_notm}} user.
+
+A typical use case would be setting up an {{site.data.keyword.IBM_notm}} watsonx platform ready to use in one or more Enterprise accounts granting administrator access
+to an AI Researcher for example.
+It enables an administrator to install all the services making up the {{site.data.keyword.IBM_notm}} watsonx platform, and automatically
+setup an {{site.data.keyword.IBM_notm}} watsonx project, so that the AI Researcher can log into the [platform](http://dataplatform.cloud.ibm.com/wx/home?context=wx) and start working.
+
+In more advanced use cases, the deployable architecture can be used as part of a larger solution, where it is included in a stack
+of other components. For example a solution that first setup the {{site.data.keyword.IBM_notm}} watsonx platform, and then
+installs a *chatbot application* that uses the underlying services.
+To facilitate those business challenges, the Watsonx AI SaaS deployable architecture provides output parameters that can be used programmatically
+for wiring it to the other components of the stack, and it provides the capability to install additional Watson services.
+
+## Architecture diagram
+{: #architecture-diagram}
+
+![Architecture diagram for the Watsonx AI SaaS Automation with Assistant and Governance deployable architecture](watsonx-saas-da.svg "Architecture diagram for the Watsonx AI SaaS Automation with Assistant and Governance deployable architecture")
+{: caption="Figure 1. Watsonx AI SaaS Automation with Assistant and Governance deployable architecture" caption-side="bottom"}{: external download="watsonx-saas-da.svg"}
+
+The Watsonx AI SaaS deployable architecture creates the services shown in the WatsonX services box and an instance of {{site.data.keyword.cos_full_notm}} in a target {{site.data.keyword.Bluemix_notm}} account, resource group, and region. Then, it automatically configures a starter project that grants access to an existing {{site.data.keyword.Bluemix_notm}} user, for example, an AI researcher. As a result, that user can
+log into the {{site.data.keyword.IBM_notm}} watsonx starter project, and start working.
+
+You can add or deploy one or more of the services from the optional services section at a later date during the lifetime of the deployable architecture.
```

Comment:
Additional watsonx services from the optional section can be installed at any time after initial deployment of the deployable architecture.